README.md

```diff
@@ -100,11 +100,11 @@ To generate a tarball archive that includes the agent binary and a configuration
 
 ### Windows
 
-To generate an MSI that will install the agent as a Windows service using a configuration file compatible with Google Cloud Monitoring:
+To build the source locally, run:
 
-1. Run `.build\msi\make.ps1 Install-Tools` to install the open source [WIX Toolset](https://wixtoolset.org).
-2. Run `.build\msi\make.ps1 New-MSI`.
-3. The MSI file will be generated in the `dist` folder.
+```
+go build  .\cmd\otelopscol\
+```
 
 Alternatively, you can generate a [googet](https://github.com/google/googet) package by running `make build-googet`. This is the packaging method used to install the Collector on Windows GCE VMs.
```

Comment:
`make build-goo` (though it's still broken, but at least matches the actual target)…

Reply:
`make build-goo` emits error:

go build github.com/GoogleCloudPlatform/opentelemetry-operations-collector/cmd/otelopscol: copying /tmp/go-build027034775/b001/exe/a.out.exe: open ./bin/google-cloud-metrics-agent_windows_amd64.exe: permission denied

sophieyfang@sophieyfang:~/opentelemetry-operations-collector/bin$ dir
google-cloud-metrics-agent_windows_amd64.exe

Reply:
FWIW, the following sequence just worked for me:
```
$ docker run -it --name ot-agent-build -v ${PWD}:/src ubuntu:focal /bin/bash
root@f70b15c99c97:/# apt-get update; apt-get -y install golang make git gettext-base
root@f70b15c99c97:/# GOBIN=/usr/local/bin go get github.com/google/googet/goopack
root@f70b15c99c97:/src# make build-goo
…
root@f70b15c99c97:/src# ls dist
google-cloud-metrics-agent.x86_64.0.0.2.goo
```

Reply:
yeah i am not worried about the copy permission error. The goal is to get the exe built and it did generate an exe as shown in my comment above.